README.md

```diff
@@ -2,35 +2,42 @@
 
 >  ⚠️ **The examples in this repository are currently work in progress** ⚠️
 
-This repository contains a couple of examples that show various aspects of the modelix platform. 
+This repository contains a couple of examples that show various aspects of the modelix platform.
 Not all features of modelix are shown here so this is not a complete demo of everything modelix can do.
-The primary focus of the examples is to show how you can use modelix to work with language and models defined in 
-MPS outside of MPS using plain Java or Kotlin. None of the examples try to reuse editors defined in MPS, if your 
-goal is to reuse MPS editors as is then modelix itself ships with a projector integration to achieve this.  
+The primary focus of the examples is to show how you can use modelix to work with language and models defined in
+MPS outside of MPS using JVM (Java/Kotlin) or web frameworks/tools.
 
-While all the examples use the same language defined in MPS they aren't meant as one complete example
-but rather as individual parts. The purpose of the examples is to give inspiration of what is possible with modelix 
+None of the examples try to reuse editors defined in MPS.
+If your goal is to reuse MPS editors as is then modelix itself ships with a projector integration to achieve this.
+
+While all the examples use the same language defined in MPS they are not meant as one complete example
+but rather as individual parts. The purpose of the examples is to give inspiration of what is possible with modelix
 and make you think and envision your own use-cases.
 
-## Repository Structure 
+## Repository Structure
+
+The repository is a single gradle project to make building everything at once easy. The examples are organized by the
+technology they use into the various sub-folders:
+
+| ID    | Components/Technologies                               | Description                                                                                                                        | Status | Folder                                                           |
+|-------|-------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------|--------|------------------------------------------------------------------|
+| `1`   | MPS + `cloud-plugin`                                  | MPS language definition that is used by all examples. The MPS "structure" is used to generate a Java API consumed by all examples. | ✅      | [mps](mps)                                                       |
```

Comment:
Why is "structure" actually quoted?

Reply:
guess as only the structure aspect is used in the `api-gen`

Reply:
I will clarify this